scripts/deployCriticalGovActions.ts

```diff
@@ -0,0 +1,172 @@
+  pauseInboxAction: string;
+  unpauseInboxAction: string;
+  pauseRollupAction: string;
+  unpauseRollupAction: string;
+  bridgeRemoveAllOutboxesAction: string;
+  bridgeAddOutboxesAction: string;
+  sequencerAddAction: string;
+  sequencerRemoveAction: string;
+}
+
+const deployAll = async () => {
+  const l1Deployer = new Wallet(l1Key, new JsonRpcProvider(l1RPC));
+
+  const hasBal = (await l1Deployer.getBalance()).gt(0);
+  if (!hasBal) {
+    throw new Error("L1 deployer has no balance");
+  }
+  const chainID = await l1Deployer.getChainId();
+  if (chainID === 5 && l2ChainID !== 421613) {
+    console.log(chainID, l2ChainID);
+
+    throw new Error("L1 / L2 missmatch");
+  } else if (chainID === 1 && l2ChainID != 42161 && l2ChainID != 42170) {
+    throw new Error("L1 / L2 missmatch");
+  }
+
+  const registryConstructorAddresses = chainIdToRegistryAddresses[l2ChainID];
+  if (!registryConstructorAddresses) {
+    throw new Error("unsupported chain id");
+  }
+
+  const connectedL1AddressRegistryFactory = new L1AddressRegistry__factory().connect(l1Deployer);
+  console.log(L1AddressRegistry__factory.name);
+
+  const l1AddressRegistry = await connectedL1AddressRegistryFactory.deploy(
+    registryConstructorAddresses.inbox,
+    registryConstructorAddresses.govL1Timelock,
+    registryConstructorAddresses.customGateway,
+    registryConstructorAddresses.l1GatewayRouter
+  );
+  await l1AddressRegistry.deployed();
+  const l1RegistryAddress = l1AddressRegistry.address;
+  console.log("deployed: L1AddressRegistry", l1RegistryAddress);
+  const factories = [
+    PauseInboxAction__factory,
+    UnpauseInboxAction__factory,
+    PauseRollupAction__factory,
+    UnpauseRollupAction__factory,
+    BridgeRemoveAllOutboxesAction__factory,
+    BridgeAddOutboxesAction__factory,
+    AddSequencerAction__factory,
+    RemoveSequencerAction__factory,
+  ];
+
+  const deploymentsAddresses: string[] = [];
+  for (let factory of factories) {
+    const contractFactory = new factory().connect(l1Deployer);
+    const contract = await contractFactory.deploy(l1RegistryAddress);
+    await contract.deployed();
+    const contractAddress = contract.address;
+    console.log("deployed:", factory.name, contractAddress);
+    deploymentsAddresses.push(contractAddress);
+  }
+  console.log(deploymentsAddresses);
+
+  const deployments: DeployedContracts = {
+    l1AddressRegistry: l1RegistryAddress,
+    pauseInboxAction: deploymentsAddresses[0],
+    unpauseInboxAction: deploymentsAddresses[1],
+    pauseRollupAction: deploymentsAddresses[2],
+    unpauseRollupAction: deploymentsAddresses[3],
+    bridgeRemoveAllOutboxesAction: deploymentsAddresses[4],
+    bridgeAddOutboxesAction: deploymentsAddresses[5],
+    sequencerAddAction: deploymentsAddresses[6],
+    sequencerRemoveAction: deploymentsAddresses[7],
+  };
+  console.log(deployments);
+
+  fs.writeFileSync(`./files/actions/critical-${l2ChainID}.json`, JSON.stringify(deployments));
```

Comment:
looks like deployment was done at point when chain IDs (in .env) were incorrect, so generated JSON files in `files/actions` have incorrect chain ID in file names. It would probably make sense to fix it manually for consistency